sst.config.ts

```diff
@@ -0,0 +1,23 @@
+import { SSTConfig } from 'sst'
+import { Tags } from 'aws-cdk-lib'
+import { LinkdexStack } from './stacks/LinkdexStack'
+
+export default {
+  config (_input) {
+    return {
+      name: 'linkdex-api',
+      region: 'us-east-2'
+    }
+  },
+  stacks (app) {
+    app.setDefaultFunctionProps({ runtime: "nodejs18.x" })
```

Comment:
```suggestion
    app.setDefaultFunctionProps({
      runtime: "nodejs18.x",
      nodejs: {
        format: 'esm',
        sourcemap: true
      }
    })
```

Reply:
https://docs.sst.dev/constructs/Function#nodejs

so that we get nice monitoring